### PR TITLE
Minor renaming

### PR DESCRIPTION
**Pull request by wcandillon**

### CODE REVIEW

File: tasks/lint.js [hunk 1]
```diff
@@ -32,4 +32,4 @@ gulp.task('lint:xqlint', function(){
         .pipe($.xqlint.failReporter());
 });
 
-gulp.task('lint:run', ['lint:jslint', 'lint:jsonlint', 'lint:xqlint']);
+gulp.task('lint', ['lint:jslint', 'lint:jsonlint', 'lint:xqlint']);
```
Comment (dknochen) [inline tasks/lint.js]: this should be moved in the main gulpfile.js then. The naming convention is that if its in a tasks file it has a prefix like this: "<file>:<function>".

**Comment by wcandillon** [comment]
Make it consistent.

**Comment by wcandillon** [comment]
@dknochen I did some cleanup. Let me know what you think.

**Comment by dknochen** [comment]
:+1:

**Pull request merged**